classifier.js

```diff
@@ -89,48 +91,44 @@ class Classifier {
 
     // Calculate all the frequences
     // word count / doc count
-    for (let i = 0; i < this.wordList.length; i++) {
-      let key = this.wordList[i];
+    this.wordList.forEach(key => {
       let word = this.dict[key];
 
-      for (let j = 0; j < this.categoryList.length; j++) {
-        let category = this.categoryList[j];
-
+      this.categoryList.forEach(category => {
         // If this word has no count for the category set it to 0
         // TODO: better place to do this or unecessary?
         if (word[category] === undefined) {
-          word[category] = {};
-          word[category].count = 0;
+          word[category] = {
+              count: 0
+            };
         }
         // Average frequency per document
-        word[category].freq = word[category].count / this.categories[category].docCount;
-      }
-    }
-
-    for (let i = 0; i < this.wordList.length; i++) {
-      let key = this.wordList[i];
+        let taxonomy = word[category];
```

Comment:
I think `wordCat` would actually be better here. Taxonomy to me is a broad term referring to the whole system of classification and this is data for one category for one word.